env/jxboot-resources/values.tmpl.yaml

```diff
@@ -70,6 +70,19 @@ gitops:
     dockerRegistryOrg: ""
 {{- end }}
 
+{{- if .Requirements.buildPacks }}
+{{- if .Requirements.buildPacks.buildPackLibrary }}
+{{- if .Requirements.buildPacks.buildPackLibrary.name }}
+    buildPackName: "{{ .Requirements.buildPacks.buildPackLibrary.name }}"
```

Comment:
Doesn't this all need to be nested under `buildPacks:`?

Reply:
No, we call it `buildPacks` within the requirements file. However everywhere else in our codebase we refer to it `buildPackName` etc. For more info there was a long discussion about it here. 

https://github.com/jenkins-x/jx/pull/7172#discussion_r424212513
https://github.com/jenkins-x/jx/issues/7126#issuecomment-628537697

Reply:
Ah, gotcha. I got confused, sorry.